towhee/pipelines/image_embedding_pipeline.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List, Union
+
+import yaml
+
+from towhee.pipelines.base import PipelineBase
+from towhee import Build, pipeline
+
+
+class ImageEmbeddingPipeline(PipelineBase):
+    """
+    Pipeline for image embedding tasks.
+
+    Args:
+        model: (`str` or `List[str]`)
+            Specifies the model used for image embedding. The user can pass a list
+            of model names to create a pipeline ensembling multiple models.
+
+            Supported models:
+                `vgg`,
+                `resnet50`, `resnet101`,
+                `swin-transformer`,
+                `vit`,
+                ...
+
+        ensemble: (`str`)
+            Specifies the type of model ensemble. This argument works iff
+            multiple model names are given via `model`.
+
+            Supported ensemble types:
+                `linear`,
+    """
+
+    def __init__(self,
+                 model: Union[str, List[str]] = None,
+                 ensemble: str = None):
+        models = []
+        if isinstance(model, str):
+            models = [model]
+        else:
+            models = model
+        num_branch = len(models)
+
+        models = dict(
+            zip([
+                'embedding_model_1',
+                'embedding_model_2',
+                'embedding_model_3',
+            ], models))
```

Comment:
it is implied that 3 embeddings ensemble is supported, if user want 4 or more, what does he need?